hms-app/src/components/HeaderMenu.tsx

```diff
@@ -18,10 +19,21 @@ export default function HeaderMenu({ links }: HeaderSearchProps) {
   const { classes } = styles()
   const today = new Date()
   today.setHours(0, 0, 0, 0)
+  const location = useLocation()
 
   const fullscreenMenu = links.map((link) => {
     return (
-      <Link key={link.label} to={link.link} className={classes.link}>
+      <Link
+        key={link.label}
+        to={link.link}
+        className={classes.link}
+        style={{
+          backgroundColor:
+            location.pathname.slice(1) === link.link
+              ? HeaderActiveColor
+              : undefined,
+        }}
+      >
```

Comment:
all of this should probably be defined in the `styles` file

Reply:
could make a new class called... headerLink or something...

Reply:
yeah but then you would to pass in location.pathname and link.link?